src/main/resources/META-INF/plugin.xml

```diff
@@ -11,6 +11,10 @@
     ]]></description>
 
     <change-notes><![CDATA[
+      <p><b>1.3.1</b></p>
+      <ul>
+        <li>Add config-file for optional github plugin dependency</li>
```

Comment:
This is not user oriented. This should be something like Plugin was not installable due to wrong plugin descriptor